src/components/ChatPanel.tsx

```diff
@@ -48,7 +48,11 @@ export interface ChatPanelProps
   extends Omit<MessageBubbleProps, "customCssClasses" | "message">,
     Omit<ChatInputProps, "customCssClasses"> {
   /** A header to render at the top of the panel. */
-  header?: JSX.Element;
+  header?: React.ReactNode;
```

Comment:
thanks! Could you run command `npm run build`? This will ensure the auto-generated documentation are up to date as well